react/features/base/media/components/AbstractAudio.js

```diff
@@ -32,7 +33,8 @@ type Props = {
      * @type {Object | string}
      */
     src: Object | string,
-    stream: Object
+    stream: Object,
+    loop?: ?boolean
```

Comment:
Are you sure, both question marks are necessary?

Reply:
I've put the first question mark for the cases when the loop prop is missing and the other one for null/undefined values.